armi/physics/neutronics/globalFlux/tests/test_globalFluxInterface.py

```diff
@@ -120,6 +119,17 @@ def test_getIOFileNames(self):
         inf, _outf, _stdname = gfi.getIOFileNames(1, 2, 1)
         self.assertEqual(inf, "armi001_2_001.GlobalFlux.inp")
 
+    def test_getHistoryParams(self):
+        params = globalFluxInterface.GlobalFluxInterface.getHistoryParams()
+        self.assertEqual(len(params), 3)
+        self.assertIn("detailedDpa", params)
+
+    def test_checkEnergyBalance(self):
```

Comment:
If possible, it would be useful to hack the underlying method to also force this method to fail, and to do an `assertRaises(ValueError)` from within this test.